sdk/python/src/dagger/engine/base.py

```diff
@@ -104,3 +87,21 @@ def get_engine(cfg: Config) -> Engine:
 
 class ProvisionError(DaggerError):
     """Error while provisioning the Dagger engine."""
+
+
+@register_engine("http")
```

Comment:
I didn't get the chance to review. It's not necessary to register this. This line should be removed.

Reply:
Hey, sorry about that, this PR was mostly just an intermediate step before the last step in this effort which is to have the SDKs all download the binary from S3 instead of pulling them from the docker image. As part of that I am deleting and refactoring tons of code across the SDKs, so I wanted to get PR in first and then clean everything up in that next one. I'll clean this up there and wait for your approval on that one 🙏